platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/DefaultProblemProgressDetails.java

```diff
@@ -277,19 +277,17 @@ public DevelocityTaskPathLocation(org.gradle.api.problems.internal.TaskPathLocat
 
         @Override
         public String getBuildPath() {
-            // TODO wolfs: What do we do here? Should we capture build path separately?
-            return ":";
+            return taskPathLocation.getBuildPath();
```

Comment:
❓ Did you consider exposing the build path the project path and the task name separately? Or is it safe to assume that we only ever need the two currently exposed here?

Reply:
We currently use build path/task path everywhere and do not use the task name separately. So from my current point of view we won't need a separate task name.